### DISCUSSION TITLE
Diffusers For TensorFlow

### DISCUSSION THREAD

**Discussion by Alok-Ranjan23** [huggingface/diffusers#7683] [Ideas]
Hi,
HuggingFace Diffusers are not available for TensorFlow. Is there any way to use HuggingFace Diffusers in TensorFlow?
Please let me know if there is any way to use HuggingFace Diffusers in TensorFlow.

Thanks,
Alok.

**Comment by tolgacangoz** [comment]
As of my knowledge, research on Diffusion models is going with PyTorch and Flax 90+%. Also, [TensorFlow seems dying](https://twitter.com/rasbt/status/1767561783382872194).
Why do you ask for TensorFlow?

**Reply by Alok-Ranjan23** [reply]
Thanks, As the transformer library has the support of Tensorflow. I thought that it should be there for the diffusers library too.
